recipes/memray/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "memray" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cfff403e1a25a61265547fe7ba696bd08bbbce93400444d66b4025cb4adc6e88
+
+build:
+  skip: True  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - python
+    - pip
+    - libunwind
+    - cython
+  run:
+    - python
+    - rich <11.0.0
+    - jinja2
+    - typing-extensions  # [py<38]
+
+test:
+  imports:
+    - memray
+  requires:
+    - pip
+  commands:
+    - pip check
+    - memray -h
+
+about:
+  home: https://github.com/bloomberg/memray
+  summary: 'Memray is a memory profiler for Python'
+  description: |
+    Memray is a memory profiler for Python. It can track memory allocations in
+    Python code, in native extension modules, and in the Python interpreter
+    itself. It can generate several different types of reports to help you
+    analyze the captured memory usage data. While commonly used as a CLI tool,
+    it can also be used as a library to perform more fine-grained profiling
+    tasks.
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```

They vendor the libbacktrace which then also needs to be represented here as it seems to be not present in the root LICENSE file see https://github.com/bloomberg/memray/blob/main/src/vendor/libbacktrace/LICENSE
```suggestion
  license: Apache-2.0 + BSD-3-Clause
  license_family: Apache
  license_file: 
    - LICENSE
    - src/libbacktrace/LICENSE
```